.github/workflows/benchmark.yml

```
@@ -73,7 +73,7 @@ jobs:
       - name: Setup .NET SDK
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 5.0.302
+          dotnet-version: 6.0.101
```

Revert benchmark build changes, this will be captured in another work item
```suggestion
          dotnet-version: 5.0.302
```